packages/documentation/copy/en/handbook-v2/Type Manipulation/Conditional Types.md

```diff
@@ -38,6 +38,46 @@ type Stuff =
 
 When the type on the left of the `extends` is assignable to the one on the right, then you'll get the type in the first branch (the "true" branch); otherwise you'll get the type in the latter branch (the "false" branch).
 
+> Be aware that this is not true for union types.
+> When the type on the left of the `extends` is a union, each of its members are evaluated for the `extends` clause independently and the results are unioned together.
+> The example below depends on this when it evaluates `number | string extends number`.
+> This is false (`number | string` is not assignable to `number`) however this type expression evaluates to the union `trueExpression | falseExpression` instead of just `falseExpression`.
+> This also applies to types that are implicitly treated like unions, for examples `boolean`: `boolean extends true` is false, but when used in a conditional type returns the union of the true and false expressions.
```

Comment:
The docs explicitly say

> The type boolean itself is actually just an alias for the union true | false.

So it's not really "implicitly treated as a union"; it *is* a union

Reply:
The Handbook is intended as a linear document, and follows the [inverted pyramid structure](https://en.wikipedia.org/wiki/Inverted_pyramid_(journalism)) of narrative wherein general information is given first, followed by more specific information later. It's also carefully structured so that a reader that doesn't understand anything about TypeScript can read it linearly, which means we can't reference terms we haven't introduced yet.

What you're describing is more of a reference page format, where all the possible relevant details are spelled out in a topically-organized manner. The reason we don't use that format for the Handbook is that you need semantic grounding for all the terms you use before it can make sense to a reader - if we're introducing the primitive types but haven't talked about union types or literal types yet, saying "boolean is a union of two literal types" is a *conceptual forward reference* to something the reader hasn't learned about yet, thus likely to confuse them.